Source/EasyNetQ/ConnectionConfiguration.cs

```diff
@@ -144,6 +144,11 @@ public ConnectionConfiguration()
     ///     Maximum channel number per connection (default is 2047)
     /// </summary>
     public ushort RequestedChannelMax { get; set; }
+
+    /// <summary>
+    ///     Per model concurrency level
+    /// </summary>
+    public int? PerModelConcurrencyLevel { get; set; } = null;
```

Comment:
Why not simply use the same name of the configuration under? ConsumerDispatcherConcurrency

Reply:
Done